cellengine/resources/fcs_file.py

```diff
@@ -225,10 +230,16 @@ def events(self, events):
     def spill_string(self):
         if self._spill_string:
             return self._spill_string
+        elif self.has_file_internal_comp:
+            self._spill_string = (
+                ce.APIClient().get_fcs_file(self.experiment_id, self._id, as_dict=True)[
+                    "spillString"
+                ]
+                or ""
+            )  # type: ignore
```

Comment:
Why is `# type: ignore` needed here? It's still string -> string.

Reply:
Because of the return type on `get_fcs_file` being `FcsFile`, but now fixed with overloads